src/test/functions.test.js

```diff
@@ -0,0 +1,15 @@
+import { countProducts, calculateComments } from '../modules/Functions.js';
+
+describe('Comments and items counter', () => {
+  test('Count Items', () => {
+    const array = ['item1', 'item2'];
+    countProducts(array);
+    expect(countProducts(array)).toEqual(2);
+  });
+
+  test('Count Comments', () => {
+    const array = ['item1', 'item2'];
+    calculateComments(array);
+    expect(calculateComments(array)).toEqual('Total Comments(2)');
+  });
+});
```

Comment:
One of the project requirements was to test the functions which calculate the count of things. You did a great job testing the `countProducts` and `calculateComments` functions. However, you forgot to test the **count for the likes**. As you have the count for the likes in the UI, kindly include the test for this functionality also.

Reply:
create a test for count likes